aws/app/lambda/organizations/organizations.js

```diff
@@ -21,128 +21,128 @@ exports.handler = async function (event) {
   /*
     Supported Methods:
     INSERT:
-      - (Required) organisationName
+      - (Required) organizationName
     GET:
-      - (Optional) organisationID. Returns all entries if not provided
+      - (Optional) organizationID. Returns all entries if not provided
     UPDATE:
       - All params are required for the first iteration of this lambda
-      - (Required) organisationID
-      - (Required) organisationNameEn
-      - (Required) organisationNameFr
+      - (Required) organizationID
+      - (Required) organizationNameEn
+      - (Required) organizationNameFr
     DELETE:
-      - (Required) organisationID to delete
+      - (Required) organizationID to delete
   */
   
   const method = event.method;
-  let organisationID = event.organisationID ? event.organisationID : null,
-    organisationNameEn = event.organisationNameEn ? event.organisationNameEn : null,
-    organisationNameFr = event.organisationNameFr ? event.organisationNameFr : null;
+  let organizationID = event.organizationID ? event.organizationID : null,
+    organizationNameEn = event.organizationNameEn ? event.organizationNameEn : null,
+    organizationNameFr = event.organizationNameFr ? event.organizationNameFr : null;
   
   let SQL = "",
     parameters = [];
   
   switch (method) {
     case "INSERT":
-      if (organisationNameEn && organisationNameFr) {
+      if (organizationNameEn && organizationNameFr) {
         // generate new uuid
-        organisationID = uuidv4();
+        organizationID = uuidv4();
         SQL = (!process.env.AWS_SAM_LOCAL) ?
-          "INSERT INTO organisations (id, nameen, namefr) VALUES (:organisationID, :organisationNameEn, :organisationNameFr) RETURNING id"
-          : "INSERT INTO organisations (id, nameen, namefr) VALUES ($1, $2, $3) RETURNING id";
+          "INSERT INTO organizations (id, nameen, namefr) VALUES (:organizationID, :organizationNameEn, :organizationNameFr) RETURNING id"
```

Comment:
We'd need to add a migration to change the table name in this case

Reply:
@Moro-Code it is part of the https://github.com/cds-snc/platform-forms-client/pull/589 pull request :) Maybe I am wrong but it looks like the migration is happening there.

Reply:
You are correct god sir ! Thanks for linking it